libs/top_level_metrics.py

```diff
@@ -128,6 +128,32 @@ def calculate_metrics_for_timeseries(
     return metrics, metric_summary
 
 
+def _calculate_smoothed_daily_cases(cases: pd.Series, smooth: int = 7):
+
+    cases = cases.copy()
+
+    # filling gaps in the series to preserve the diff of the first
+    # value after the gap. For example, a cumulative series of [10, None, 30] would result
+    # in a diff of [10, None, None], dropping any signal from the last value.
```

Comment:
I removed the comment, but I think this is something to think about - I don't think this case logic is quite right as is, but is closer.  Since I'm not relying on the case values on the frontend just yet, I'd like to merge this and iterate on a future PR.